azure-template-tox-job.yml

```diff
@@ -18,7 +18,7 @@ jobs:
   timeoutInMinutes: 0
   pool:
     ${{ if eq(parameters.os, 'macos') }}:
-      vmImage: 'macOS-10.15'
+      vmImage: 'macOS-11'
```

Comment:
Thanks for pointing this out to me—I'll need to go through and update our other projects too.